### PR TITLE
CEDS-1720 Content for DMSREJ error messages

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: app/services/model/RejectionReason.scala [hunk 1]
```diff
@@ -62,6 +64,9 @@ object RejectionReason {
     }.getOrElse(Seq.empty)
   }
 
+  private def logMissingPointerMessageKey(pointer: Option[Pointer])(implicit messages: Messages): Unit =
```
Comment (ed0906) [inline app/services/model/RejectionReason.scala]:
```
error.pointer.filter { p =>
 val defined = messages.isDefinedAt(p.messageKey)
 if(!defined) {
    logger.warn("...")
 }
 defined
}
```

Comment (ed0906) [reply app/services/model/RejectionReason.scala]: Not much different but saves traversing the pointer list twice.

Up to you which you prefer

Comment (PatrykRudnicki) [reply app/services/model/RejectionReason.scala]: Good point

Comment (PatrykRudnicki) [reply app/services/model/RejectionReason.scala]: Fixed

**Pull request merged**